<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogHighlight</class>
 <widget class="QDialog" name="DialogHighlight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DialogHighlight</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>370</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>51</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>51</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textBoxName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>320</width>
     <height>21</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelColor">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>51</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>51</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>Color:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPickerColor">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>57</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>41</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxSearchWord">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>360</width>
     <height>180</height>
    </rect>
   </property>
   <property name="title">
    <string>Search Words</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>90</width>
      <height>140</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonAdd">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRemove">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="listSearchWords">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>250</width>
      <height>140</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>270</width>
      <height>9999</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBoxFontWeight">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Thin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ExtraLight</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Light</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Normal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Medium</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DemiBold</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bold</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ExtraBold</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Black</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelFontWeight">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>51</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>Font Weight:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogHighlight</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogHighlight</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
